README.md

```diff
@@ -109,11 +119,18 @@ The generated source code is a spring boot rest maven project with a read me fil
 
 To build and run   :
 
+## With Linux
+
 ```
 cd testServer-0.0.1/
 mvn clean package -Dmaven.test.skip=true
 mvn spring-boot:run -Dmaven.test.skip=true
 ```
+## With Windows (example as editor "Spring Tools Suite")
+
+    import project to the editor as maven project 
+    install PostgreSQL 
+    run as spring boot application
```

Comment:
either you add as list of simple text, in both case , you need to respect indentation.
```suggestion
- import project to the editor as maven project 
- install PostgreSQL 
- run as spring boot application
```